client/my-sites/checkout/checkout/source-payment-box.jsx

```diff
@@ -134,6 +136,28 @@ class SourcePaymentBox extends PureComponent {
 		} );
 	}
 
+	renderBankOptions() {
+		const idealBanks = {
+			'-': translate( 'Select bank…' ),
+			abn_amro: 'ABN AMRO',
+			asn_bank: 'ASN Bank',
+			bunq: 'Bunq',
+			ing: 'ING',
+			knab: 'Knab',
+			rabobank: 'Rabobank',
+			regiobank: 'RegioBank',
+			sns_bank: 'SNS Bank',
+			triodos_bank: 'Triodos Bank',
+			van_lanschot: 'Van Lanschot',
+		};
+
+		return map( idealBanks, ( text, optionValue ) => (
```

Comment:
What about we remove the `-` entry from the object and do it like this?
```
	return (
		<option value="-" key="-">
			{ translate( 'Please select your bank…' ) }
		</option>
		{ map( idealBanks, ( text, optionValue ) => (
			<option value={ optionValue } key={ optionValue }>
				{ text }
			</option>
		)}
	);
```

Reply:
👍